x-pack/filebeat/module/cisco/asa/_meta/fields.yml

```diff
@@ -85,3 +85,13 @@
     type: short
     description: >
       ICMP code.
+
+  - name: connection_type
+    type: keyword
+    description: >
+      The VPN connection type
+
+  - name: dap_records
+    type: keyword
+    description: ->
```

Comment:
```suggestion
    description: >
```

Reply:
my bad

> This looks good to me. Just a minor comment on fields.yml.
> 
> Can you please add an entry to the CHANGELOG.next.asciidoc file under the Added/Filebeat section.
> 
> From checking the Travis CI output, it looks like you need to run `make update` from the filebeat and/or x-pack/filebeat directories to re-generate some files.

My bad with the wrong "-", I changed them and made a "make update" in the /beats/filebeat/xpack and /beats/filebeat folder